be/src/exec/json_scanner.cpp

```diff
@@ -105,7 +105,9 @@ StatusOr<ChunkPtr> JsonScanner::get_next() {
 
     if (src_chunk->num_rows() == 0) {
         if (status.is_end_of_file()) {
-            return Status::EndOfFile("EOF of reading json file, nothing read");
+            // NOTE: can not stop right here because could be more files to read.
+            // return Status::EndOfFile("EOF of reading json file, nothing read");
```

Comment:
remove line 109?

Reply:
I want to keep this line here for comment.  let more people know here could be a big mistake.